cmd/kube-controller-manager/app/apps.go

```diff
@@ -50,6 +50,7 @@ func startDaemonSetController(ctx context.Context, controllerContext ControllerC
 
 func startStatefulSetController(ctx context.Context, controllerContext ControllerContext) (controller.Interface, bool, error) {
 	go statefulset.NewStatefulSetController(
+		ctx,
```

Comment:
Let's add a name for the controller here:

```suggestion
		klog.NewContext(ctx, klog.LoggerWithName(klog.FromContext(ctx), "statefulset"))
```